bin/compile

```diff
@@ -106,6 +124,7 @@ export EJSON_PRIVATE_KEY=$(cat "$ENV_DIR/EJSON_PRIVATE_KEY")
 heading "Sourcing environment variables on compile"
 source $BUILD_DIR/.profile.d/export_ejson_secrets.sh
 
-if [[ "${EJSON_DECRYPT_FAILED}" == "true" ]]; then
+export_ejson_secrets
+if [ $? -ne 0 ]; then
```

Comment:
I [prefer double square brackets over single](https://google.github.io/styleguide/shell.xml?showone=Test,_%5B_and_%5B%5B#Test,_%5B_and_%5B%5B) as they are less likely to introduce errors